### DISCUSSION TITLE
Basis support inside KTX2 container

### DISCUSSION THREAD

**Discussion by vorg** [visgl/loaders.gl#1272] [Q&A]
Is there plan in enabling basis transcoder in KTX Loader inside CompressedTextureLoader in order to support basis in gltf files as per [KHR_texture_basisu](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu)?

**Comment by ibgreen** [comment]
Short answer is yes. That said, we are focusing on basis support in `I3SLoader` at the moment, which doesn't involve glTF, but the required glTF changes do not look big. 

Let's see if we can get this into ongoing 3.0 development (which we plan to release towards end of April). Can't guarantee at the moment but it makes a lot of sense.

PRs are welcome :)
